charts/k8s-shredder/templates/cluster-role-binding.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.rbac.create}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "k8s-shredder.fullname" . }}
```

Comment:
Why `template` in some places instead of `include`?

Reply:
No reason, let me switch to `include` everywhere